api/numaresourcesoperator/v1/numaresourcesoperator_types.go

```diff
@@ -85,7 +85,7 @@ type NodeGroupConfig struct {
 	// InfoRefreshPeriod sets the topology info refresh period. Use explicit 0 to disable.
 	// +optional
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Topology info refresh period setting",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
-	InfoRefreshPeriod *metav1.Duration `json:"infoRefreshPeriod,omitEmpty"`
+	InfoRefreshPeriod *metav1.Duration `json:"infoRefreshPeriod,omitempty"`
```

Comment:
this is a fix which should be backport ASAP, should be extracted from this PR, merged and backported ASAP

Reply:
PR created https://github.com/openshift-kni/numaresources-operator/pull/696

Will rebase over main as soon as merged.